tidy3d/web/core/http_util.py

```diff
@@ -7,13 +7,18 @@
 
 import requests
 import toml
-from tidy3d.web.cli.constants import CONFIG_FILE
 
+from .constants import (
+    SIMCLOUD_APIKEY,
+    SIMCLOUD_VERSION,
+    KEY_APIKEY,
+    HEADER_APIKEY,
+    KEY_VERSION,
+    HEADER_VERSION,
+)
+from ..cli.constants import CONFIG_FILE
```

Comment:
if this is importing from CLI, I suppose we will have the CLI be an MC controlled directory?

Reply:
modified